Fix the OpenShift GitOps init command

mprahl opened the pull request:
Some users report that the other command didn't work. This new approach
seems to work for everyone.

This also changes it so that the image tags use the `{product-version}`
variable.

Relates:
https://github.com/stolostron/backlog/issues/24747

mprahl committed:
Fix the OpenShift GitOps init command

Some users report that the other command didn't work. This new approach
seems to work for everyone.

This also changes it so that the image tags use the `{product-version}`
variable.

Signed-off-by: mprahl <mprahl@users.noreply.github.com>

dockerymick reviewed (approved):
Looks good, approving based on related PR: https://github.com/stolostron/rhacm-docs/pull/3777#pullrequestreview-1057258880

dockerymick reviewed (changes requested):
Hold off on merging this, the conref didnt work

mprahl committed:
Instruct the user to set the version when installing the PG

The `{product-version}` variable does not get replaced in the source
YAML, so instruct the user to replace the version.

mprahl commented:
I updated the PR with the changes in #3779.

dockerymick committed:
Update policy_generator.adoc

small change
